cart-item-controller.js

```diff
@@ -0,0 +1,25 @@
+const {getCartItemsByCartItemId} = require('../services/cart-item-service');
+
+const getCartItemsByCartItemIdRoute = (server) => {
+	server.route({
+		path: '/cart-items/{cartItemId}',
+		method: 'GET',
+		handler: (request, h) => {
+			const cartItem = getCartItemsByCartItemId(request.params.cartItemId);
```

Comment:
```suggestion
			const cartItem = getCartItemByCartItemId(request.params.cartItemId);
```

we want a single resource by single resource id.